docs/modules/ROOT/pages/documentation/ecosystem/payara-jax-rs-ssl.adoc

```diff
@@ -0,0 +1,36 @@
+= Payara JAX-RS Client Automatic SSL Context
+
+JAX-RS Client Automatic SSL Context is an implementation to automatically indicate the certificate when doing a Rest Client call
+
+To enable this we need to include the next dependency on the application
+
+[source,XML]
+----
+<dependency>
+  <groupId>com.ecosystem.extensions</groupId>
+  <artifactId>ecosystem-rest-ssl-configuration</artifactId>
+  <version>1.0</version>
+</dependency>
```

Comment:
Artefact location obviosuly pending -  we don't own `com.ecosystem` coordinates.